features/breol_news/breol_news.features.field_instance.inc

```diff
@@ -333,9 +385,10 @@ function breol_news_field_default_field_instances() {
 
   // Translatables
   // Included for use with string extractors like potx.
+  t('Brødtekst');
   t('Cover Image');
-  t('Indhold');
   t('Karouseller');
+  t('Manchet');
```

Comment:
Should both be named the same ('Brødtekst' vs 'Manchet') or?

Reply:
No, one should be named “Manchet“ and one should be named “Brødtekst”.